packages/blockchain/src/airdrop/checkSablierAirdropEligibility.ts

```diff
@@ -0,0 +1,101 @@
+import { MerkleTree } from 'merkletreejs';
+import type { Address } from 'viem';
+import { keccak256, encodeAbiParameters, parseAbiParameters } from 'viem';
+
+import { getPublicClient } from '../getPublicClient';
+
+// @ts-ignore
+import sablierMerkleInstantAbi from './SablierMerkleInstant.json' assert { type: 'json' };
```

Comment:
maybe try "resolveJsonModule": true in tsconfig.base.json

Reply:
didn't work, I had to manually include the json file in the src field of tsconfig.json for blockchain package